content/coordination/onboarding/index.en.md

```diff
@@ -111,8 +111,8 @@ In the Slack, together with all R-Ladies chapters organizers worldwide,
 you will find the
 R-Ladies Global leadership team. Feel free to reach out to them through
 the channel #ask_the_leadership if you have questions that are not
-addressed in the Starter Kit, the Guidelines, or that other organizers
-cannot answer. More info about R-Ladies Global team here:
+addressed in the Starter Kit, or the Guidelines, or that other organizers
```

Comment:
```suggestion
addressed in the Guidelines or that other organisers
```

Starter-Kit no longer really contains relevant information, so we shouldnt mention it.